assembl/static2/js/app/utils/routeMap.js

```diff
@@ -35,7 +35,8 @@ const routes = {
   theme: 'theme/${themeId}',
   administration: '${slug}/administration',
   unauthorizedAdministration: '${slug}/unauthorizedAdministration',
-  adminPhase: '${phase}'
+  adminPhase: '${phase}',
+  post: '${slug}/debate/${phase}/theme/${themeId}/#${element}'
```

Comment:
Maybe to be more clear, you could rename `element` to `elementId` to be consistent with the `themeId` that is just before it.